test/byok_test.go

```diff
@@ -17,6 +17,8 @@ func (s *CLITestSuite) TestBYOK() {
 		// delete tests
 		{args: "byok delete cck-001", input: "y\n", fixture: "byok/delete_1.golden"},
 		{args: "byok delete cck-404", fixture: "byok/delete_2.golden", exitCode: 1},
+		{args: "byok delete cck-002 cck-006 cck-007 cck-100", fixture: "byok/delete_3.golden", exitCode: 1},
```

Comment:
I've never been a huge fan of numbering integration tests... they always get out of order, and they're not descriptive.
```suggestion
		{args: "byok delete cck-002 cck-006 cck-007 cck-100", fixture: "byok/delete-multiple.golden", exitCode: 1},
```

Reply:
Is this testing that one of these IDs doesn't exist?

Reply:
> Is this testing that one of these IDs doesn't exist?
Yup. I'll do a pass at the end and rename all the number-name files I introduced. I agree that that naming scheme is not good.